internal/server/server.go

```diff
@@ -32,18 +30,14 @@ const DefaultWriteTimeout = 2 * time.Minute
 // Server defines parameters for running an HTTP server.
 type Server struct {
 	logger log.Logger
-	prober *prober.Prober
 	srv    *http.Server
 
 	opts options
 }
 
 // New creates a new Server.
 func New(logger log.Logger, reg *prometheus.Registry, opts ...Option) Server {
-	options := options{
-		gracePeriod: DefaultGracePeriod,
```

Comment:
It's ok to remove the flag, but I think we should consider specifying a default value. At least `120s` it's the default for our queries to timeout.